packages/ckeditor5-build-classic/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@ckeditor/ckeditor5-build-classic",
+  "name": "ckeditor5-custom-build",
```

Comment:
Why did you rename it to classic?

Reply:
It is not me. It is done by whoever set  up the ckeditor first. I just have to reintroduce this changes.
I am not happy this this approach either. How can reform CKEditor superbuild set up to the proper way? I mean forking CKEditor repo is overkill. The superbuild stuff should be done in another manner according CKEidtor docs - https://ckeditor.com/docs/ckeditor5/latest/installation/getting-started/quick-start-other.html#creating-custom-builds-with-online-builder